src/components/banner/_banner.scss

```diff
@@ -75,68 +73,67 @@ $logo-breakpoint: 1400px;
   &__logo {
     width: 3.3em;
 
-    @media screen and (min-width: 90em) {
+    @media screen and (min-width: map-get($breakpoint, "breakpoint-10")) {
       left: 2em;
       top: 3em;
     }
 
-    @media screen and (min-width: $logo-breakpoint) {
+    @media screen and (min-width: map-get($breakpoint, "breakpoint-10")) {
       position: fixed;
       z-index: 1000;
 
       svg {
         opacity: 0;
         transform: translateY(50%);
-        transition:
-          opacity 400ms ease,
-          transform 400ms ease;
+        transition: opacity $duration-4 $timing-ease,
+          transform $duration-4 $timing-ease;
       }
 
       &--visible {
         svg {
           opacity: 1;
-          transform: translateY(0)
+          transform: translateY(0);
         }
       }
     }
   }
 
   &__button {
-    background: rgba(53, 54, 61, .3);
-    border: 0;
-    border-radius: 50%;
+    background: rgba(53, 54, 61, 0.3);
+    border: $border-none;
+    border-radius: $border-radius-half;
     cursor: pointer;
     height: 2em;
     padding: 0;
     position: relative;
-    transition: background .2s ease-in-out;
+    transition: background 0.2s $timing-ease-in-out;
```

Comment:
This shouldn't have a trailing 0. Can can we use or add to our variables rather than hard coding it?